testnet-contracts/contracts/Recoverer.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.5.0;
+
+// TODO: Refactor Recoverer to ECDSA based signature validation
+contract Recoverer {
```

Comment:
Is there an OpenZeppelin or widely used version of this contract that we can use for functionality rather than implement our own?

Reply:
Yes, available for v0.5.0 as a library: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/cryptography/ECDSA.sol. Will complete the TODO by swapping it in for the Recoverer contract.